src/Concerns/FiltersQuery.php

```diff
@@ -70,7 +70,7 @@ protected function ensureAllFiltersExist()
 
         $diff = $filterNames->diff($allowedFilterNames);
 
-        if ($diff->count()) {
+        if ($diff->count() && ! config('query_builder.disable_invalid_filter_query_exception')) {
```

Comment:
I would suggest splitting the check for the configuration flag into a separate if-statement with an early return at the beginning of this method.
This would make it easier to parse mentally and save some operations at runtime.

Reply:
Good suggestion, I'll change it after merging :)